recipes/fontconfig/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+import os
+from conans import ConanFile, CMake, tools, RunEnvironment
+
+class FontconfigTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        bin_path = os.path.join("bin", "example")
+        env = {"FONTCONFIG_PATH": os.path.join(self.deps_cpp_info["fontconfig"].rootpath, "etc", "fonts")}
```

Comment:
You're removing the `etc` folder in `package`, so is this working?
Should this folder be installed elsewhere?

Should this be moved to `package_info` as a `self.env_info.FONTCONFIG_PATH = ...`?